requirements.txt

```diff
@@ -1,7 +1,7 @@
 bx-python>=0.8.10
 guppy3>=3.1.0
 matplotlib>=3.3.4
-pandas>=1.2.3
+pandas>=1.1
```

Comment:
But why was this necessary? There were no issues with `pip` installation previously (?)

Reply:
Yes, I run into this, but I was not upgrading pip. I will reverse this.

Reply:
OK, done. Works fine.